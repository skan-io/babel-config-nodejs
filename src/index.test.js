import config from '.';

test('babel-config regression', ()=> {
  expect(config).toMatchSnapshot();
});
