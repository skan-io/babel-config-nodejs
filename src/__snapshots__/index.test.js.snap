// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-config regression 1`] = `
Object {
  "env": Object {
    "test": Object {
      "presets": Array [
        Array [
          "@babel/preset-env",
          Object {
            "targets": Object {
              "node": "current",
            },
            "useBuiltIns": false,
          },
        ],
      ],
    },
  },
  "plugins": Array [
    "dynamic-import-node",
    "@babel/plugin-syntax-dynamic-import",
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-numeric-separator",
    "@babel/plugin-proposal-optional-catch-binding",
    "@babel/plugin-proposal-throw-expressions",
    "@babel/plugin-transform-runtime",
  ],
  "presets": Array [
    Array [
      "@babel/preset-env",
      Object {
        "targets": Object {
          "node": "10",
        },
        "useBuiltIns": false,
      },
    ],
  ],
}
`;
